packages/service-clients/odsp-client/src/interfaces.ts

```diff
@@ -87,6 +91,11 @@ export interface OdspContainerServices {
 	 * Provides an object that facilitates obtaining information about users present in the Fluid session, as well as listeners for roster changes triggered by users joining or leaving the session.
 	 */
 	audience: IOdspAudience;
+
```

Comment:
Are users expected to ever construct or implement the `OdspContainerServices` interface? Or do we exclusively provide objects of this type to them? If the latter, we should consider marking the interface as `@sealed` so that we can add new members in the future without it being considered a breaking change.

Reply:
I think we return this interface in createContainer or getContainer call and very likely we need to add new stuff to this interface for ODSP specific features. I don't think clients need to implement this interface. @vladsud for any comments.
I will try to mark it as sealed.

Reply:
This is type of out parameter in our API, and we expect no custom implementations.
That said, it's totally normal to construct custom implementation as a mock in tests.
I think sealed is fine.